Neos.Media.Browser/Resources/Public/JavaScript/select.js

```diff
@@ -12,17 +12,39 @@ window.addEventListener('DOMContentLoaded', (event) => {
 									if (localAssetIdentifier !== '') {
 											window.parent.NeosMediaBrowserCallbacks.assetChosen(localAssetIdentifier);
 									} else {
-										$.post(
-											$('link[rel="neos-media-browser-service-assetproxies-import"]').attr('href'),
-													{
-															assetSourceIdentifier: $(this).attr('data-asset-source-identifier'),
-															assetIdentifier: $(this).attr('data-asset-identifier'),
-															__csrfToken: $('body').attr('data-csrf-token')
-													},
-													function(data) {
-															window.parent.NeosMediaBrowserCallbacks.assetChosen(data.localAssetIdentifier);
-													}
+										if ($(this).attr('data-import-in-process') !== 'true') {
```

Comment:
Can you please when you anyway update the code also replace jQuery code with Vanilla JS ❤️